README.md

```diff
@@ -12,7 +12,7 @@ Once you've done some hacking and you'd like to have your work merged, you'll ne
 
 ## Prerequisites
 
-### Linux
+### Debian/Ubuntu
```

Comment:
These package names aren't correct for Debian/Ubuntu. They're just the library names.